opencti-platform/opencti-front/lang/front/de.json

```diff
@@ -2691,9 +2686,16 @@
   "entity_Tracking-Number": "Tracking-Nummer",
   "entity_Credential": "Berechtigung",
   "Auto collapse submenus in left navigation": "Untermenüs in der linken Navigation automatisch einklappen",
-  "All add or remove actions done on the graph after the last expansion will be lost.": "Alle Hinzufügungs- oder Entfernungsaktionen, die nach der letzten Erweiterung am Diagramm vorgenommen wurden, gehen verloren.",
```

Comment:
doesn't seem right to loose this translation, maybe a rebase issue ?

Reply:
Yes it seems a lot of translations have been removed... I'll check all of them manually